sla/metrics.py

```diff
@@ -102,14 +102,14 @@ def get_containers_healthcheck(host):
         return 1
 
     json = response.json()
-    if json['res'] != 1:
-        for error in response.json()['errors']:
-            logger.info(error)
+    if json['error'] is not None:
```

Comment:
it is possible that we will have json without field `error`?
also - redefining `json` name

Reply:
corrected